### PR TITLE
BAH-412 Setup i18n for owa UI

### PR DESCRIPTION
**Pull request by sanishmaharjan**
BAH-412: Implement internationalization (I18N) in Bed Management UI
ReOpen with resolving Travis-CI build fail issue.
Please code review @mks-d

### CODE REVIEW

File: owa/app/index.js [hunk 1]
```diff
@@ -1,8 +1,16 @@
...
+import {addLocaleData} from 'react-intl';
 import App from './components/app';
 import {BrowserRouter} from 'react-router-dom';
 
+import en from 'react-intl/locale-data/en';
+import es from 'react-intl/locale-data/es';
+import fr from 'react-intl/locale-data/fr';
+import it from 'react-intl/locale-data/it';
+import pt from 'react-intl/locale-data/pt';
+addLocaleData([...en, ...es, ...fr, ...it, ...pt]);
```
Comment (mks-d) [inline owa/app/index.js]: So this kind of piece of code will be part of what must be changed in the JIRA ticket to come, correct?

File: owa/app/utilities/localeHelper.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+            allowedLocales: response.data.allowedLocales ? response.data.allowedLocales : ['en']
+        };
+    };
+
+    getLocaleCode = (locale) => {
+        return locale.toLowerCase().split(/[_-]+/)[0];
+    };
+
+    getNativeNameByLocaleCode = (locale) => {
+        return i18nIosCodes.getNativeName(locale);
```
Comment (mks-d) [inline owa/app/utilities/localeHelper.js]: `i18nIosCodes` → `i18nIsoCodes`

Comment (sanishmaharjan) [reply owa/app/utilities/localeHelper.js]: @mks-d Done.

File: owa/app/index.js (hunk 1 above)
Comment (sanishmaharjan) [reply owa/app/index.js]: Yes, this would be tricky to import dynamically.

Comment (mks-d) [reply owa/app/index.js]: In the JIRA ticket to come, we will advocate for not using those `react-intl` libraries, but rather to have the `Session` REST endpoints return complete locales descriptions.

**Pull request merged**